mosip_master/data_upgrade/1.1.5.5_to_1.2.0.1/upgrade.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+properties_file="$1"
+echo `date "+%m/%d/%Y %H:%M:%S"` ": $properties_file"
+if [ -f "$properties_file" ]
+then
+    echo `date "+%m/%d/%Y %H:%M:%S"` ": Property file \"$properties_file\" found."
+    while IFS='=' read -r key value
```

Shellcheck (0.11.0)

[warning] value appears unused. Verify use (or export if used externally).

(SC2034)